pkg/component/seedsystem/seedsystem.go

```diff
@@ -104,11 +108,26 @@ func (s *seedSystem) WaitCleanup(ctx context.Context) error {
 
 func (s *seedSystem) computeResourcesData() (map[string][]byte, error) {
 	var (
+		excessCapacityConfigs []gardencorev1beta1.SeedSettingExcessCapacityReservationConfig
+
 		registry = managedresources.NewRegistry(kubernetes.SeedScheme, kubernetes.SeedCodec, kubernetes.SeedSerializer)
 	)
 
 	if s.values.ReserveExcessCapacity.Enabled {
-		if err := s.addReserveExcessCapacityDeployment(registry); err != nil {
+		defaultConfig := gardencorev1beta1.SeedSettingExcessCapacityReservationConfig{
+			// This roughly corresponds to a single, moderately large control-plane.
+			Resources: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("2"),
+				corev1.ResourceMemory: resource.MustParse("6Gi"),
+			},
+		}
+		excessCapacityConfigs = append(excessCapacityConfigs, defaultConfig)
+	}
+	excessCapacityConfigs = append(excessCapacityConfigs, s.values.ReserveExcessCapacity.Configs...)
```

Comment:
Reading https://github.com/gardener/gardener/issues/8307#issuecomment-1665278498 again,
Do we still need the default config if a custom config is present?

Reply:
Nope, we don't, it is just another config. We could deprecate the flag in seed config and remove it later.
WDYT?

Reply:
Makes sense, operators can use the current default as custom config if they want.

Reply:
`.spec.settings.excessCapacityReservation.enabled` is deprecated now.
Additionally, I updated the defaulting to use the new structure.

Reply:
Now that the `Enabled` field is deprecated, we should always add the configs, not only `if s.values.ReserveExcessCapacity.Enabled` ?

Reply:
In combination with the defaulting you would have an double reservation and you could not disable it.

Reply:
Ah, The looping happens below, sorry, I got confused.